apis/filesystem/src/main/java/org/jclouds/filesystem/util/Utils.java

```diff
@@ -100,12 +102,39 @@ public static void delete(File file) throws IOException {
    }
 
    /**
+    * @return Localized name for the "Everyone" Windows principal.
+    */
+   public static final String getWindowsEveryonePrincipalName() {
+      if (isWindows()) {
+         try {
+            Process process = new ProcessBuilder("whoami", "/groups").start();
+            try {
+               String line;
+               try (BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()))) {
+                  while ((line = reader.readLine()) != null) {
+                     if (line.indexOf("S-1-1-0") != -1) {
+                        return line.split(" ")[0];
+                     }
+                  }
+               }
+            } finally {
+               process.destroy();
+            }
+         } catch (IOException e) {
+         }
+      }
+      return "Everyone";
```

Comment:
Fine to merge as-is - just a minor about perhaps commenting this as a fallback and/or default value?